manifests/dotnet.yml

```diff
@@ -454,7 +454,7 @@ tests/:
     test_config_consistency.py:
       Test_Config_Dogstatsd: missing_feature (does not support hostname)
       Test_Config_RateLimit: v3.4.1
-      Test_Config_Tags: missing_feature
+      Test_Config_Tags: v3.12.0
```

Comment:
Just curious, do we need to wait for a release to enable .NET system tests? Is there a way to specific a "dev" package for dd-trace-dotnet releases?

Reply:
I'm not sure. The dd-trace-dotnet repo has its `master` branch set to the next release version, so the in-development version is the same as the next released version. I'll look into this for my next set of system-tests.

@cbeauchesne do you have any prior knowledge that could help here?

BTW this is non-blocking since this version has in fact already been released so I'll continue to merge the PR but this comment thread is still useful.